src/index.js

```diff
@@ -2,18 +2,16 @@ import express, { json } from 'express';
 import cors from 'cors';
 import morgan from 'morgan';
 import swagger from 'swagger-ui-express';
-import swaggerDoc from '../documentation/swaggerDoc.json';
 import { config } from 'dotenv';
+import swaggerDoc from '../documentation/swaggerDoc.json';
 import router from './routers/index';
-import bodyParser from 'body-parser';
```

Comment:
Any reason behind the removal of `body-parser`

Reply:
Express now has body-parse by efault so there is no longer a need to install body-parser(app.json()) is now enough for body-parser